src/middlewares/verifyJwtToken.js

```diff
@@ -0,0 +1,50 @@
+import JwtService from 'lesgo/src/services/JwtService';
+import ErrorException from 'Exceptions/ErrorException';
+import config from 'Config/jwt';
+
+const token = headers => {
+  if (!headers.Authorization) {
+    throw new ErrorException('Authorization Header is required!');
+  }
+
+  const parsed = headers.Authorization.split(' ');
+
+  if (!parsed[1]) {
+    throw new ErrorException('Missing token!');
+  }
+
+  return parsed[1];
+};
+
+const verifyJwtToken = () => {
+  return {
+    before: (handler, next) => {
+      const { headers } = handler.event;
+
+      try {
+        const service = new JwtService(token(headers), config.secret, {
```

Comment:
The client does not need to know the technical details to verifying jwt token. Instead, it should be as simple as a single line validation. I.e;

```
import { validateJwtToken } from 'lesgo';

const verifyJwtToken = () => {
  return {
    ...
    return validateJwtToken(token, config.secret, config.options);
  }
}
```

Reply:
Applied above

Reply:
Applied above